rca/static_src/sass/components/_booking-bar.scss

```diff
@@ -111,4 +111,23 @@
             }
         }
     }
+
+    .app--short-course & {
```

Comment:
It looks like we really have custom margin bottoms depending on the page based on the code above.